src/Layouts/Partials/Artboard.vala

```diff
@@ -47,7 +47,7 @@ public class Akira.Layouts.Partials.Artboard : Gtk.ListBoxRow {
 
 	public Artboard (Akira.Window main_window, string name) {
 		Object (
-			window: main_window, 
```

Comment:
Again, please revert this file. It doesn't have any changes other than indentation and we should keep the blame history as clean as posible :)